test/uk/gov/hmrc/taxhistory/services/EmploymentHistoryServiceSpec.scala

```diff
@@ -161,6 +161,11 @@ class EmploymentHistoryServiceSpec extends UnitSpec with MockitoSugar with TestU
       await(testEmploymentHistoryService.retrieveRtiData(testNino, TaxYear(2016))) shouldBe Some(testRtiData)
     }
 
+    "stub rti employments succeeds with a None" in
```

Comment:
Trivial: could rename this test to be consistent with the other tests below on lines 184 & 189